openrtx/include/core/graphics.h

```diff
@@ -238,6 +241,18 @@ point_t gfx_printBuffer(point_t start, fontSize_t size, textAlign_t alignment,
 point_t gfx_print(point_t start, fontSize_t size, textAlign_t alignment,
                   color_t color, const char* fmt, ... );
 
+/**
+ * Prints symbols the screen at the specified coordinates.
+ * @param start: text line start point, in pixel coordinates.
+ * @param size: text font size, defined as enum.
+ * @param alignment: text alignment type, defined as enum.
+ * @param color: text color, in color_t format.
+ * @param fmt: printf style format string
+ * @return text width and height as point_t coordinates
+ */
+point_t gfx_printSymbol(point_t start, fontSize_t size, textAlign_t alignment,
```

Comment:
From what I understand from the code, symbols are bound to text characters. What if we use an enum to identify the symbol, instead?